### DISCUSSION TITLE
Manual Mesh Leveling - Manual Tuning

### DISCUSSION THREAD

**Discussion by wbvczar** [Jyers/Marlin#1865] [Q&A] [answered]
I am running this latest version of Jyers firmware on my Ender 3 v2.  Under the Level menu, there is an option for "Manual Tuning".  I use this function after creating a new Mesh.  After I print the leveling calibration print, I then go into "Manual Tuning" and adjust at each point to fine tune the mesh.  I love this option as I don't have to re-create the entire mesh.

My question is; can I enable this on my Ender 3 Pro?  I have upgraded the board to the 4.2.7 board and I still have the original LCD screen.  Is there something to enable in the Configuration.h file?  Is this even possible on the original LCD screen?  I have enabled Mesh Bed Levelling on the 3 Pro, but I don't have the "Manual Tuning" option.

**Comment by Cal-LF** [comment, marked as answer]
@wbvczar The E3 pro LCD screen is not supported. If you changed it to a E3V2 screen, it should work.

**Reply by wbvczar** [reply]
Thank you!
